README.md

```diff
@@ -1416,12 +1416,12 @@ Currently being reworked as Neo Launcher.
 
 ## Streaming
 ### CloudStream
-<img alt="CloudStream" height="64" src="https://github.com/LagradOst/CloudStream-3/blob/master/app/src/main/ic_launcher-playstore.png?raw=true">
+<img alt="CloudStream" height="64" src="https://apt.izzysoft.de/fdroid/repo/com.lagradost.cloudstream3/en-US/icon.png">
```

Comment:
change image to https://raw.githubusercontent.com/recloudstream/cloudstream/master/app/src/main/ic_launcher-playstore.png

Reply:
done